ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Hello! Thanks for clicking that shiny new issue button.
+
+We want you to get help when you need it and opening an issue here will probably just slow you down, so could you do us a favor? Go check README.md and curators.md to make sure this is the most appropriate place for what you're about to post. We have much faster ways of solving individual problems, and they're linked to there.
+
+...
+
+Still here? Great! Just a little bit of work will make sure we hear your voice and solve your problem in the long term. Could you please fill out the sections below? Just follow the directions in each one, and delete these instructions when you're done.
+
+We may or may not respond to your issue, but we appreciate you taking the time to open it! Issues usually have to "bake" for a while so things can be solved the right way. Check out that talk linked in README.md for why.
+
+Finally, thanks for making the Elm community a wonderful place to be!
+-->
+
+## Context (What I'm Trying To Accomplish)
+
+<!--
+Describe in broad details what you're trying to do. We're looking for "I'm trying to create a music visualizer" over "I can't work with the Web Audio API". More context is better. Set the scene!
+-->
+
+## Specifics (What's Not Working)
+
+<!--
+Here's where you can get specific and crunchy. What have you tried? What workarounds have failed? Give as many specific details as you can.
```

Comment:
fair. That's jargon from another group I'm in that snuck in here. I'll remove it.